include/rocksdb/statistics.h

```diff
@@ -434,6 +434,20 @@ enum Tickers : uint32_t {
   BLOCK_CHECKSUM_COMPUTE_COUNT,
   MULTIGET_COROUTINE_COUNT,
 
+  // Integrated BlobDB specific stats
+  // # of times cache miss when accessing blob from blob cache.
+  BLOB_DB_CACHE_MISS,
+  // # of times cache hit when accessing blob from blob cache.
+  BLOB_DB_CACHE_HIT,
+  // # of data blocks added to blob cache.
+  BLOB_DB_CACHE_ADD,
+  // # of failures when adding blobs to blob cache.
+  BLOB_DB_CACHE_ADD_FAILURES,
+  // # of bytes read from blob cache.
+  BLOB_DB_CACHE_BYTES_READ,
+  // # of bytes written into blob cache.
```

Comment:
Nit: can we replace `written` with `added` just to avoid confusion?

Reply:
Done; Many thanks to @riversand963 for code review.